rules/catch-error-name.js

```diff
@@ -47,18 +36,36 @@ const create = context => {
 	const stack = [];
 
 	function push(value) {
-		if (stack.length === 1) {
-			stack[0] = true;
+		stack.push(value);
+	}
+
+	function indexifyName(name, scope) {
```

Comment:
Would be better to reuse https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/rules/utils/avoid-capture.js here instead. See: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/257 // @futpib

Reply:
Ping @doniyor2109

Reply:
This was done as part of https://github.com/sindresorhus/eslint-plugin-unicorn/pull/306 which is almost certainly what is causing the merge conflict.